cli/dcoscli/task/main.py

```diff
@@ -161,6 +174,156 @@ def _log(follow, completed, lines, task, file_):
     return 0
 
 
+def get_nested_container_id(task):
+    """ Get the nested container id from mesos state.
+
+    :param task: task definition
+    :type task: dict
+    :return: comma separated string of nested containers
+    :rtype: string
+    """
+    assert isinstance(task, dict)
```

Comment:
not a fan of the assert. this should be validated before the method and if thats not valid output, should give the user an error about what part was not as expected